qiita_db/analysis.py

```diff
@@ -100,11 +100,14 @@ def get_by_status(cls, status):
         set of int
             All analyses in the database with the given status
         """
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT analysis_id FROM qiita.{0} a JOIN qiita.{0}_status ans "
-               "ON a.analysis_status_id = ans.analysis_status_id WHERE "
-               "ans.status = %s".format(cls._table))
-        return {x[0] for x in conn_handler.execute_fetchall(sql, (status,))}
+        with TRN:
+            sql = """SELECT analysis_id
```

Comment:
This indentation is odd, and this SQL should also be changed to use `USING`. Standard SQL indentation is:
```SQL
SELECT analysis_id  FROM qiita.{0}
JOIN qiita.{0}_status USING (analysis_status_id)
WHERE ans.status = %s
```